.github/workflows/docs.yaml

```diff
@@ -0,0 +1,26 @@
+name: docs
+on:
+  push:
+#    branches: [master] # @todo uncomment this
+jobs:
+  generate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Read .nvmrc
+        run: echo ::set-output name=NVMRC::$(cat .nvmrc)
+        id: nvm
+      - name: Use Node.js ${{ steps.nvm.outputs.NVMRC }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '${{ steps.nvm.outputs.NVMRC }}'
+      - name: Install dependencies
+        run: npm ci
+      - name: Generate documentation
+        run: npm run jsdoc
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
```

Comment:
@cedpoilly, this will push the generated documentation (in the `./docs` folder) to a `gh-pages` branch in practice to serve the documentation via GitHub Pages. I don't have admin access to this repo, but we'll want to set this up in repository settings before merging:

<img width="611" alt="Screen Shot 2020-08-15 at 9 17 18 AM" src="https://user-images.githubusercontent.com/847532/90313166-51820880-ded8-11ea-9388-56e180da6988.png">